backend/app/core/scraper_service.py

```
@@ -0,0 +1,354 @@
+            async def background_all_scraping_task():
+                try:
+                    start_time = datetime.now()
+                    results = await self.registry.execute_all_scrapers(max_pages=max_pages, concurrent=concurrent)
+                    processing_time = (datetime.now() - start_time).total_seconds()
+                    
+                    # Process results
+                    total_scraped = sum(r.scraped_events for r in results if isinstance(r, ScraperResult))
+                    total_saved = sum(r.saved_events for r in results if isinstance(r, ScraperResult))
+                    
+                    # Create comprehensive result
+                    multi_result = MultiScrapeResponse(
```

Ruff (0.12.2)

Local variable `multi_result` is assigned to but never used

Remove assignment to unused variable `multi_result`

(F841)